CHANGELOG.md

```diff
@@ -1,3 +1,12 @@
+## [1.7.0]
```

Comment:
I don't think 1.6.3 has been released, so 1.6.3 and 1.7.0 should be combined?

Reply:
oh, did i not actually release that

Reply:
ok updated

Reply:
honestly this changelog is pointless

Reply:
I tend to agree, but the course has been set.